SConstruct

```diff
@@ -65,7 +65,7 @@ def scons():
 
     env = Environment(TOOLS=['extra', 'default'])
 
-    opts_file = os.path.join(Dir('#').abspath, 'daos_m.conf')
+    opts_file = os.path.join(Dir('#').abspath, 'daos.conf')
```

Comment:
we might want to have some temporary code 

if os.path.exists("daos_m.conf"):
     os.rename("daos_m.conf", "daos.conf") before this to avoid messing up existing environments